packages/app/ios/RNFBApp/RNFBAppModule.m

```diff
@@ -205,8 +205,7 @@ - (void)invalidate {
         firApp = [FIRApp appNamed:appName];
       }
     } @catch (NSException *exception) {
-      // TODO js error builder
-      // reject(@"code",@"message",[exception ]);
+      return [RNFBSharedUtils rejectPromiseWithExceptionDict:reject exception:exception];
```

Comment:
🙈 good spot

Reply:
This was the root of a user issue so kudos to them really, I just traced through till I ended up here and was happy it was obvious!